src/Components/Admin/Rooms/roomCard.js

```diff
@@ -233,16 +242,19 @@ export default function RoomCard({
                     >
                         Share & Publish
                     </MenuItem>
-                    <MenuItem disabled className={classes.menuItem}>
-                        View Game Stats
-                    </MenuItem>
                     <Divider light />
-                    <MenuItem disabled className={classes.menuItem}>
+                    <MenuItem
+                        className={classes.menuItem}
+                        onClick={() => {
+                            setAnchorEl(null);
+                            navigator.clipboard.writeText(
+                                process.env.REACT_APP_ADMIN_DEPLOYED_URL +
+                                    `admin/environment/${data.id}`
```

Comment:
At the moment this is the production url, so if you are testing from local (as you will see in my loom video), you will need to log in again since it is a different website entirely.